openasip/doc/man/OpenASIP/OpenASIP.tex

```diff
@@ -4136,6 +4136,61 @@ \subsection{Defining a custom operation using an HDL snippet}
 but same information should still be printed to stdout meaning that the algorithm
 was executed correctly.
 
+\subsection{QEMU-OpenASIP}
+Simulating custom RISC-V instructions for larger programs may be too slow for 
```

Comment:
Something like this could make it more accurate: 
"Using RTL simulation for verifying and debugging software running in OpenASIP designs with custom RISC-V instructions may be too slow, especially with larger programs. To support faster early software development and debugging, OpenASIP supports integration with QEMU: The base QEMU RISC-V model is used for non-customized instructions and when the QEMU encounters an instruction it doesn't recognize, OpenASIP emulates it using the OSAL definitions.

You can build the OpenASIP QEMU fork with:"


Try to avoid "our" etc. possessive terms. It's open source, not owned by anyone :)